src/pages/Repos.tsx

```diff
@@ -0,0 +1,32 @@
+import { Divider, Spin, Row, Col } from 'antd'
+import '@styles/RepoCard.styles.scss'
+import { RepoCard } from '@components'
+import { GithubRepoContent } from '@types'
+import { useGetRepos } from '@services'
+
+export const Repos = () => {
+  const { isLoading, isError, data } = useGetRepos()
+
+  if (isError) {
+    return <div>Error fetching repositories.</div>
+  }
+  if (isLoading) return <Spin size='large' style={{ margin: 'auto' }} />
+
+  return (
+    <div style={{ width: '100%' }}>
```

Comment:
it is bad design to have inline styles, kindly use className instead

Reply:
Refactored this to have `full-width` class